clientd3d/d3drender.c

```diff
@@ -926,14 +924,12 @@ void D3DRenderBegin(room_type *room, Draw3DParams *params)
 		D3DCacheFlush(&gWorldCacheSystem, &gWorldPool, 1, D3DPT_TRIANGLESTRIP);
 
 		IDirect3DDevice9_SetRenderState(gpD3DDevice, D3DRS_STENCILENABLE, FALSE);
-		IDirect3DDevice9_SetRenderState(gpD3DDevice, D3DRS_ZENABLE, D3DZB_TRUE);
+		IDirect3DDevice9_SetRenderState(gpD3DDevice, D3DRS_ZENABLE, TRUE);
 		if (gD3DDriverProfile.bFogEnable)
 			IDirect3DDevice9_SetRenderState(gpD3DDevice, D3DRS_FOGENABLE, TRUE);
 
-		// restore the view and world matrices
-		MatrixIdentity(&mat);
+		// restore the view matrix
 		IDirect3DDevice9_SetTransform(gpD3DDevice, D3DTS_VIEW, &view);
-		IDirect3DDevice9_SetTransform(gpD3DDevice, D3DTS_WORLD, &mat);

```

Comment:
I don't understand why this was removed---what does the world matrix have to do with translucency?